content/docs/started/k8s/kfctl-istio-dex.md

```diff
@@ -37,6 +37,11 @@ Configuring your installation with {{% config-file-istio-dex %}} has a few optio
   1.3.1 with SDS enabled, which requires you to use Kubernetes 1.13 or later. You may need
```

Comment:
Can we make the wording more precise, where it says "You may need to add...". As a reader, my question here is, under what circumstances to I need to add the extra configs? 

For example, would the following wording be clearer?
```
Make sure that your configuration includes the following settings:
```